js/movie_browser.js

```diff
@@ -1,2 +1,74 @@
-// API Docs at: 
-// http://www.omdbapi.com
\ No newline at end of file
+// API Docs at:
+// http://www.omdbapi.com
+$( document ).ready(function() {
+  console.log( "ready!" );
+});
+
+//on form submit this function runs two other functions
+function myFunction1(){
+  myFunction();
+  runFunction();
+}
+
+//changes the div display
+function myFunction(){
+
+  $("#movie_select").css("display", "inline-block");
```

Comment:
yap I did define it in my stylesheet as " display: none; ", here it's being displayed. so this just change it from "none" to "inline-block."